### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by barshathakuri**
## Addresses:
- #114 

## Changes
- Add read me
- Add API doc
- Add issue template

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,85 @@
...
+        └── RootLayout/
+            └── index.tsx (Defines root layout and requests fetched for DomainContext)
+```
+
+## TODO
+
+- [ ] Roadmap
+- [ ] Contributing
+- [ ] License
+- [ ] Contact
```
Comment (samshara) [inline README.md]: Let's remove this section.

File: README.md [hunk 2]
```diff
@@ -1,87 +1,85 @@
...
+    ├── contexts/ (Custom contexts)
+    ├── declarations/ (Type declarations for external libraries)
+    ├── hooks (React Hooks)
+    │   └── domain/ (Hooks that are specific to a domain)
+    ├── index.tsx (Initializes React)
+    ├── utils (Utility functions)
+    │   └── domain/ (Utility functions that are specific to a domain)
+    └── views/ (Pages that we can navigate on the platform)
+        └── RootLayout/
+            └── index.tsx (Defines root layout and requests fetched for DomainContext)
```
Comment (samshara) [inline README.md]: This doesn't seem to be out directory structure. Let's use actual directory structure.

**Review by samshara — CHANGES_REQUESTED** [review]

File: README.md
Comment (samshara) [inline README.md:1]: Also, a nice resource to learn about the README.md file and its usage and how to write one.
https://github.com/hackergrrl/art-of-readme

**Review by AdityaKhatri — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report---staging.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+*If applicable, add screenshots to help explain your problem.*
+
+## Expected behaviour
+*Please describe how this feature should be working.*
+
+## Related feature
+*Please cross-reference the build item to which this relates, if known*
+
+## Criticality/Urgency
+*Is it critical that this ticket is resolved urgently (e.g. is it a Blocker)?  What is the impact if no change is made?*
```
Comment (samshara) [inline .github/ISSUE_TEMPLATE/bug-report---staging.md]: Do we have staging environment? If not maybe we don't need this.

File: APIDOC.md
Comment (samshara) [inline APIDOC.md:1]: Let's rename this to APIDOCS.md

File: APIDOC.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+       "altair-graphql-plugin-graphql-explorer"
+    ]
+    ```
+
+For more information, you can visit [altair plugins](https://altairgraphql.dev/docs/plugins/).
+
+## Project Configuration
+
+### Add Schema
+* Go to the [alert-hub-backend](https://github.com/IFRCGo/alert-hub-backend) repository and download the [schema.graphql](https://github.com/IFRCGo/alert-hub-backend/blob/develop/schema.graphql) file
```
Comment (samshara) [inline APIDOC.md]:
```suggestion
* Go to the [alert-hub-backend](https://github.com/IFRCGo/alert-hub-backend) repository and download the [schema.graphql](https://github.com/IFRCGo/alert-hub-backend/blob/develop/schema.graphql) file.
```

File: APIDOC.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+*Note: The schema file might change in the future. Make sure to use the latest one*
```
Comment (samshara) [inline APIDOC.md]: Let's move this to a new line.

File: APIDOC.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+### Add GraphQL URL
+* Go to the [alert-hub](https://alert-hub.westeurope.cloudapp.azure.com/map) staging platform and inspect the page
+* Go to the `Network` tab and click **Fetch/XHR** button
+![image](https://s3-ap-southeast-1.amazonaws.com/tc-codimd/uploads/a575a073f6aa81839c739a208.png)
+
+* Click on **graphql** and copy the request URL in the `Header` tab
+![image](https://s3-ap-southeast-1.amazonaws.com/tc-codimd/uploads/a575a073f6aa81839c739a209.png)
+
+* Paste the request URL in the `Enter URL` text field
+![image](https://s3-ap-southeast-1.amazonaws.com/tc-codimd/uploads/a575a073f6aa81839c739a20a.png)
```
Comment (samshara) [inline APIDOC.md]: Why are we doing this long process to get the URL. Shouldn't we just have listed down the URL?

**Review by samshara — DISMISSED** [review]

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**